sphinx_immaterial/apidoc/python/section_titles.py

```diff
@@ -1,10 +1,15 @@
 import docutils.nodes
 import docutils.statemachine
 
+import sphinx
 from sphinx.domains.python import PyObject
 import sphinx.util.nodes
 
 
+# Not needed in Sphinx 5.3
+assert sphinx.version_info < (5, 3)
```

Comment:
Wouldn't this break usage with Sphinx prior to v5.3?

Is this file not imported elsewhere (making `assert` ok)?

Reply:
This is only imported from `sphinx_immaterial/apidoc/python/default.py` where I've added a check of `sphinx.version_info`.